app/src/docs/v1.api-spec.yaml

```diff
@@ -15,19 +16,27 @@ servers:
 security:
   - OpenID: []
 paths:
-  /checks/status:
+  /health:
     get:
       summary: Returns status of correspondent APIs
       operationId: getChecksStatus
```

Comment:
Should we rename this operationId?

Reply:
Good catch. It's been renamed to getHealth to be consistent with the other operationIds.